pkg/cloud/gcp/actuators/util/gcp_uefi_disk_check_test.go

```diff
@@ -0,0 +1,339 @@
+package util_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	machinev1 "github.com/openshift/api/machine/v1beta1"
+	computeservice "github.com/openshift/machine-api-provider-gcp/pkg/cloud/gcp/actuators/services/compute"
+	"github.com/openshift/machine-api-provider-gcp/pkg/cloud/gcp/actuators/util"
+)
+
+var _ = Describe("IsUEFICompatible", func() {
```

Comment:
This suite would be better written as a table case rather than using `JustBeforeEach` I think, wanna give that a go?